views/Chart.react.js

```diff
@@ -1,9 +1,21 @@
 import React from 'react'
+import Button from '@material-ui/core/Button';
+import basePathConfig from '../server/configs/basePathConfig';
+
+const basePath = basePathConfig || '';
 
 export default function Charts() {
   return (
-    <div>
+    <>
       <h1> Charts</h1>
-    </div>
+      <br />
+      <Button
+        variant="outlined"
+        color="primary"
+        href={`${basePath}/charts/new`}
```

Comment:
can we extract the routes to a routes object so we can reference this with `routes.newChart`?

Reply:
[89c25f1](https://github.com/AMP-SCZ/dpdash/pull/78/commits/89c25f103294e3e583dd2ca8d95164bd04b81388)